css/styles.css

```diff
@@ -660,6 +650,113 @@ form {
   border: none;
 }
 
+/* Mobile Menu starts  */
+.mobile-menu-wrapper {
+  max-height: 787px;
+  min-width: 375px;
+  background-color: #6070ff;
+  mix-blend-mode: multiply;
+  z-index: 100;
+  display: none;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  margin-top: 80px;
+  backdrop-filter: blur;
```

Comment:
- Great job for fixing the hamburger button issue raised by the previous reviewer :+1: 

- [ ] Your design is slightly different from the one in Figma. Kindly check the screenshots below:

| Yours | Figma |
| --- | --- |
| ![Screenshot from 2022-09-13 22-53-07](https://user-images.githubusercontent.com/63087491/190019941-7e441a3b-9ee6-4b01-9c20-827ad03f2d17.png) | ![Screenshot from 2022-09-13 23-00-34](https://user-images.githubusercontent.com/63087491/190019969-1c49971c-537f-4d95-abf8-79ba7ceecb4e.png) |

Let's make sure that the font weight, and size are the same as in Figma. Also, adjust your close icon to position as seen in the original screenshot. Your contact option is hidden by the header text. Let's fix the issues. Also, ensure that there are no console errors.